routes/user.js

```diff
@@ -426,4 +428,54 @@ router.get('/contacts', auth, async (req, res) => {
   }
 });
 
+const storage = multer.memoryStorage();
+const upload = multer({ storage });
+
+router.post('/avatar', auth, upload.single('file'), async (req, res) => {
+  const file = req.file;
+  const userId = req.user.id;
+  const s3FileUrl = process.env.AWS_S3_FILE_URL;
+
+  try {
+    const user = await User.findById(userId);
+
+    if (!user) {
+      return res.status(404).send('User not found');
+    }
+
+    let s3bucket = new AWS.S3({
+      accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+      secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+      region: process.env.AWS_REGION,
+    });
+
+    const params = {
+      Bucket: process.env.AWS_BUCKET,
+      Key: file.originalname,
```

Comment:
what happens if two diff users uploads the same filename? would this overwrite each other?